web/src/components/tasks/RaffleBingoPaginator.tsx

```diff
@@ -0,0 +1,162 @@
+import { Content } from "@/components/Content";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { CtaContainer } from "@/components/njwds-extended/cta/CtaContainer";
+import { SingleCtaLink } from "@/components/njwds-extended/cta/SingleCtaLink";
+import { HorizontalStepper } from "@/components/njwds-extended/HorizontalStepper";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+import { ActionBarLayout } from "@/components/njwds-layout/ActionBarLayout";
+import {
+  getAgencyName,
+  getRaffleBingoTask,
+  RaffleBingoSteps,
+  shouldDisplayBackButton,
+  shouldDisplayContinueButton,
+  shouldDisplayMarkAsCompleteButton,
+} from "@/components/tasks/RaffleBingoSteps";
+import { TaskStatusChangeSnackbar } from "@/components/TaskStatusChangeSnackbar";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { Task } from "@/lib/types/types";
+import { TaskProgress } from "@businessnjgovnavigator/shared/userData";
+import { ReactElement, ReactNode, useEffect, useState } from "react";
+
+interface Props {
+  task: Task;
+}
+
+export const RaffleBingoPaginator = (props: Props): ReactElement => {
+  const [stepIndex, setStepIndex] = useState(0);
+  const [task, setTask] = useState<Task>();
+  const { business, updateQueue } = useUserData();
+  const [successSnackbarIsOpen, setSuccessSnackbarIsOpen] = useState<boolean>(false);
+
+  const { Config } = useConfig();
+
+  useEffect(() => {
+    getRaffleBingoTask(stepIndex).then((task) => {
+      setTask(task);
+    });
```

Comment:
[sand] For the same reason as above.
```suggestion
    getRaffleBingoTask(stepIndex).then((step) => {
      setStep(step);
    });
```